Assets/LeapMotion/Editor/CustomEditorBase.cs

```diff
@@ -118,6 +119,19 @@ public class CustomEditorBase : Editor {
       }
     }
 
+    /// <summary>
+    /// Defer rendering of a property until the end of the inspector.  Deferred properties
+    /// are drawn in the REVERSE order they are deferred!  NOT by the order they appear in the 
```

Comment:
Great example of this is the slider class, which contains all of it's events at the bottom in a nice order thanks to the reverse order they are added into the deferred list.